packages/invoices/src/OutboundShipment/DetailView/DetailView.tsx

```diff
@@ -16,7 +16,7 @@ import { getOutboundShipmentDetailViewApi } from '../../api';
 import { GeneralTab } from './tabs/GeneralTab';
 import { ItemDetailsModal } from './modals/ItemDetailsModal';
 
-import { ItemRow } from './types';
+import { InvoiceLineRow } from './types';
```

Comment:
as you'll see below, I've renamed `ItemRow` to `InvoiceLineRow` because `ItemRow` was an old name and wasn't very representative of what it was. Though, `InvoiceLineRow` isn't better.

Maybe head towards something like:
`InvoiceNode` -> `Invoice` -> `OutboundShipment`
`InvoiceLineNode` -> `InvoiceLine` -> `OutboundShipmentLine`

Where the `XNode` are GQL types, `Invoice` are simple 'mappings' into the more simple forms, and `OutboundShipmentX` can be the more specific type with methods attached and any extra details?

Reply:
that sounds good!